change/@ni-nimble-components-9da58386-e822-45d9-a191-56fc2717b88b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I tested in Edge and it successfully prunes the accessibility tree. In Firefox's accessibility tree, however, I still see the buttons and they have a warning saying that clickable elements must be focusable. I found [many many Mozilla issues](https://bugzilla.mozilla.org/show_bug.cgi?id=aria-hidden) about this. It seems there has been a decade-long debate about whether it's correct to remove these from the tree. I couldn't actually figure out what they concluded, if anything. Some of the issues seem to suggest they are now supposed to be pruned, but that's not happening for me.

I'm mostly mentioning this for posterity and in case you want to dig in further. If not I think it's OK to move forward with this as-is since it improves behavior in Chromium browsers (Safari doesn't have a dev tools a11y tree so I didn't test there).